### DISCUSSION TITLE
Perhaps replace iOS lazer file system with Apple’s “Choose a File” system when importing files?

### DISCUSSION THREAD

**Discussion by littlemanthe1** [ppy/osu#17261] [Ideas] [answered]
As it is, I don’t think it’s possible to access files stored on something like your downloads or iCloud, with the file directory being stuck at the root directory, possibly requiring jailbreaking your iPhone or iPad in order to import files to lazer
![DA385C98-74DB-4302-93D7-BD34933CEB01](https://user-images.githubusercontent.com/22322776/158431777-7d7b0f8a-4329-430c-a659-3fa963660412.jpeg)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/20581
